frontend/shared-components/data-table/data-table.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from "react"
+
+import { useTable } from "react-table"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faGreaterThan, faPlusCircle } from "@fortawesome/free-solid-svg-icons"
+
+import styles from "./data-table.module.css"
+
+type IncidentData = {
+  dates: string
+  incidentType: string
+  officersInvolved: string[]
+  subject: string
+  source: string
+}
+
+export function DataTable({ count = 220375 }) {
+  // Defines table rows
+  const { dTable, dHeader, dFooter, dPagContainer, dRowPage, dTableWrapper } = styles
+  const [rowsShown, setRowsShown] = React.useState(7)
+
+  // TODO: When this gets changed from mocking to fetching the data from an api call, append
+  // the full and save values dynamically
+  const data = React.useMemo(
+    () => [
+      {
```

Comment:
That changes is forthcoming, particularly since i need a longer mock to test pagination.

Reply:
Oh? I'm curious why it would require a React component at all? I'm just talking about a simple array object variable you could plug in directly where we have the opening/closing square brackets lines 18-145

(Also so sorry I forgot to check back in on this so long -- got into design stuff and forgot to check back in with PR reviews)

Reply:
Oh yeah, you're right, I just need to import the array I'm feeding into the useMemo calls 🤡